src/openrct2/rct1/S4Importer.cpp

```diff
@@ -117,18 +119,18 @@ class S4Importer final : public IParkImporter
     EntryList _waterEntry;
 
     // Lookup tables for converting from RCT1 hard coded types to the new dynamic object entries
-    uint16_t _rideTypeToRideEntryMap[RCT1_RIDE_TYPE_COUNT]{};
-    uint16_t _vehicleTypeToRideEntryMap[RCT1_VEHICLE_TYPE_COUNT]{};
-    uint16_t _smallSceneryTypeToEntryMap[256]{};
-    uint16_t _largeSceneryTypeToEntryMap[256]{};
-    uint16_t _wallTypeToEntryMap[256]{};
-    uint16_t _pathTypeToEntryMap[24]{};
-    uint16_t _pathAdditionTypeToEntryMap[16]{};
-    uint16_t _sceneryThemeTypeToEntryMap[24]{};
+    ObjectEntryIndex _rideTypeToRideEntryMap[RCT1_RIDE_TYPE_COUNT]{};
+    ObjectEntryIndex _vehicleTypeToRideEntryMap[RCT1_VEHICLE_TYPE_COUNT]{};
+    ObjectEntryIndex _smallSceneryTypeToEntryMap[256]{};
+    ObjectEntryIndex _largeSceneryTypeToEntryMap[256]{};
+    ObjectEntryIndex _wallTypeToEntryMap[256]{};
+    ObjectEntryIndex _pathTypeToEntryMap[24]{};
+    ObjectEntryIndex _pathAdditionTypeToEntryMap[16]{};
+    ObjectEntryIndex _sceneryThemeTypeToEntryMap[24]{};
```

Comment:
Should we convert all these hardcoded numbers to constants in the future?

Reply:
Not sure. They're local and only used for RCT1 import.